app/shared/taginput/taginput.component.html

```diff
@@ -1,13 +1,13 @@
 ﻿<div>
-    <tag-input [(ngModel)]="usernames"
-                [onlyFromAutocomplete]="true"
-                [autocompleteItems]="names"
-                [autocomplete]="true"
-                (onRemove)="onRemove($event)"
-                (onAdd)="onAdd($event)"
-                [placeholder] ="tagPlaceholder | translate"
-                [maxItems]="maxItems"
-                [secondaryPlaceholder]="placeholder"
-                (keyup)="updateUserList($event)" ngDefaultControl>
-    </tag-input>
+  <tag-input [(ngModel)]="usernames"
```

Comment:
The text of the is displayed below and not in the box - `height: unset;` applied to `delete-icon svg` would fix this.